src/quo2/components/wallet/account_overview/component_spec.cljs

```diff
@@ -0,0 +1,77 @@
+(ns quo2.components.wallet.account-overview.component-spec
+  (:require [quo2.components.wallet.account-overview.view :as account-overview]
+            [test-helpers.component :as h]
+            [utils.i18n :as i18n]))
+
+(h/describe "Account overview test"
+  (h/test "renders correct account name"
+    (h/render [account-overview/account-overview
+               {:account-name "Diamond Hand"}])
+    (-> (h/get-by-text "Diamond Hand")
+        h/is-truthy))
+
+  (h/test "renders correct account value"
+    (h/render [account-overview/account-overview
+               {:current-value "20.0$"}])
+    (-> (h/get-by-text "20.0$")
+        h/is-truthy))
+
+  (h/test "renders correct account changes"
+    (h/render [account-overview/account-overview
+               {:currency-change   "€0.00"
+                :percentage-change "0.00%"}])
+    (and
+     (-> (h/get-by-text "0.00%")
```

Comment:
do we need this `->` here?
won't `(h/is-truthy (h/get-by-text "0.00%"))` be sufficient here? same for most of these tests? 🤔